catalystwan/api/mtt_aaa_api.py

```diff
@@ -156,19 +167,26 @@ class TenantTacacsAPI:
     def __init__(self, session: ManagerSession) -> None:
         self.session = session
         self.url_path = "/dataservice/admin/tacacs"
-        self.tenant_id = self.session.get_tenant_id()
 
     def __str__(self) -> str:
         return str(self.session)
 
+    @property
+    def tenant_org_name(self):
+        """
+        Get the tenant org name
+        :return:
+        """
+        return self.endpoints.configuration_settings.get_organizations().first().org
+
     @status_ok
     def add_tacacs(self, tacacs_server: TenantTacacsServer):
         """
         Create TACACS for tenant
         :param tacacs_server:
         :return:
         """
-        logger.debug(f"TACACS config tenant_id={self.tenant_id}.")
+        logger.debug("TACACS config {self.tenant_org_name}")
```

Comment:
Are you aware that `add_tacacs` method will make two requests to vManage each time it is invoked?
`GET /settings/configuration/organization`
`POST /dataservice/admin/tacacs`
Why we want to mix tacas API with settings API? I think this is unnecessary coupling.

Reply:
These methods can be easily decorated outside SDK if specific logging is needed.